wcivf/apps/elections/templates/elections/includes/_person_card.html

```diff
@@ -30,6 +31,10 @@ <h6 class="ds-candidate-name ds-h5">
                 {% endif %}
             </p>
         {% endif %}
+
+        {% if person_post.previous_party_affiliations.all %}
```

Comment:
`previous_party_affiliations` is a field on the `PersonPost` model. So why isn't this working?

Reply:
Yes, I have. Using that data, I have the screen shots above for ballot and person views, but this is for the post code view and it's not rendering the data like the others.